client/src/pages/record.tsx

```diff
@@ -116,6 +119,11 @@ function RecordPage(props: {
       navigate("/");
     }
   }
+  function onEnd() {
```

Comment:
let's move `refetchMentor` to be managed by the `useWithRecordState` hook

Reply:
Talked to Aaron and it doesn't look like we need to refetch the mentor live anymore. Removed this instead.